policy/policy.sample.json

```diff
@@ -0,0 +1,71 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
+                    "SegmentID": "01CA06A0-429B-4EE1-9185-F17D5CE4D0F2",
+                },
+                {
+                    "Tenant":"tenant1",
+                    "Segment": "Segment2",
+                },
+            ],
+            "Tags": [
+                {
+                    "Key": "Platform",
+                    "Value": "Openstack",
+                },
+                {
+                    "Key": "TenantID",
+                    "Value": "D46FD18B-A381-4297-A6D4-9DA594EE71C8",
+                },
+            ], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Egress", # can be Egress or Ingress.
+            "Rules": [
+                {
+                    "CidrBlock": "0.0.0.0/0", # IP from L3 header
```

Comment:
Hold on. Statefulness is not on the rule level (I just noticed that)? But statefulness is most closely associated with the protocol you wish to handle, which is expressed in the rules. We can have rules for so many different protocols here, some of which may not even have a support for statefulness in iptables. I would suggest that the stateful flag should go into the rule.
 
So, I can see a use case where someone wishes to create a policy, which is applicable to a particular complex application. That application may require a few different TCP connections, some UDP and maybe a few other protocols. Some need to be handled in a stateful manner, some not. So you create a rule set for that application.

Now you deploy this application in different places. So the same rule set (policy) needs to be applied to (associated with?) different tenants/segments.

Of course, every time you deploy this policy the "peer" may change as well. For example, you may bundle your application with the policy. So, depending on who deploys your application and where it is deployed, the "peer" changes as well.

Therefore, the peer cannot be part of the rules, if we wish for the rule set to be reusable.